### PR TITLE
Event desktop layout

### PR DESCRIPTION
**Pull request by gestchild**
References #1790

## Type
✨ Feature  

## Value
Uses CSS grids to layout template

## Screenshot

localhost_3002_events_wil_diqaacqahlu6 1

localhost_3002_events_wil_diqaacqahlu6

- [x] Browser tested: `npm run test:browsers <URL>`
- [x] Works without JS in the client

### CODE REVIEW

File: client/gulpfile.js [hunk 1]
```diff
@@ -142,7 +142,8 @@ gulp.task('scss:compileCritical', () => {
       browsers: [
         'last 2 versions',
         'iOS 8'
-      ]
+      ],
+      grid: false
```
Comment (jamesgorrie) [inline client/gulpfile.js]: I forgot to ask - do you have a screenshot of what this looks like in IE?

File: client/scss/grids/_css_grid.scss [hunk 2]
```diff
@@ -1,16 +1,51 @@
...
+  @include clearfix;
+
+  @supports(display: grid) {
+    display: grid;
+    grid-auto-rows: minmax(min-content, max-content);
+
+    @each $label, $map in $grid-config {
+      $respond: map-get($map, 'respond');
+      $min: map-get($map, 'padding');
+      $columns: '[full-start] minmax(#{$min}, 1fr) [main-start] minmax(0, #{$container-width-max - (map-get($map, 'padding') * 2)}) [main-end] minmax(#{$min}, 1fr) [full-end]';
```
Comment (jamesgorrie) [inline client/scss/grids/_css_grid.scss]: could it be useful to leave a comment of an example property string?

File: client/scss/grids/_css_grid.scss [hunk 3]
```diff
@@ -1,16 +1,51 @@
...
+
+      @include respond-to(nth($respond, 1)) {
+        grid-template-columns: unquote($columns);
+      }
+    }
+  }
+}
+
+@mixin max-width-fallback {
```
Comment (jamesgorrie) [inline client/scss/grids/_css_grid.scss]: As everything is global - could we call it `max-width-grid-fallback`?

File: client/gulpfile.js (hunk 1 above)
Comment (gestchild) [reply client/gulpfile.js]: screen shot 2018-01-08 at 11 13 26

**Pull request merged**